.circleci/config.yml

```diff
@@ -17,24 +17,24 @@ jobs:
 
       - run:
           name: scalastyle
-          command: sbt scalastyle test:scalastyle
+          command: sbt -v -mem 1024 scalastyle test:scalastyle
```

Comment:
We could probably avoid repetition by utilising the `environment` section:

```
jobs:
  build:
    environment:
      - SBT: "sbt -v -mem 1024"
    steps:
      - run:
          name: scalastyle
          command: $SBT scalastyle test:scalastyle
```

Also note that `-mem` does not necessarily force the amount:

```
$ SBT="sbt -v -mem 1024"
$ $SBT scalastyle
[process_args] java_version = '1.8.0_112'
# Executing command line:
java
-XX:ReservedCodeCacheSize=128m
-Xmx4096m
-Xms512m
-Xss64m
-XX:MaxMetaspaceSize=1024m
-jar
/usr/local/Cellar/sbt/0.13.12/libexec/sbt-launch.jar
scalastyle
```

Reply:
With regard to memory it seems the priority is:
1. `JAVA_OPTS` var
2. `sbtopts` file
3. `mem` switch

That's good, it means we can still override this memory setting within Circle's web UI

Reply:
I implemented the repetition avoidance suggestion.